MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include *.rst *.md LICENSE
+recursive-include typeshed *.py
+recursive-include typeshed/stdlib *.pyi
+recursive-include typeshed/third_party *.pyi
```

Comment:
Can't the latter two be combined?

Reply:
Those directories are symlinks and I couldn't get it to work with just one line. Globbing inside those directories works fine though.

Reply:
OK